src/popup/components/AppsBrowserListItem.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="apps-browser-list-item-wrapper">
```

Comment:
I think this should be a `<button>` or `<a>` as we are watching for `click` event on the parent level.